src/v2/Apps/Artists/Components/ArtistsLetterNav.tsx

```diff
@@ -6,13 +6,17 @@ import { RouterLink, RouterLinkProps } from "v2/Artsy/Router/RouterLink"
 
 export const LETTERS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ".split("")
 
-interface ArtistsLetterNavProps extends BoxProps {}
+interface ArtistsLetterNavProps extends BoxProps {
+  inDropDown?: boolean
+}
```

Comment:
Rather than do this, it looks like I forgot to spread the rest props on the Flex there. Just add the {...rest} to that element then you can just pass any justifyContent you want from the call site.